Automattic/jetpack
# E2E tests: use Allure steps

#### Changes proposed in this Pull Request:

Include test steps in Allure test report

#### Jetpack product discussion

1156386383419466-as-1200559827847716

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:
* [Check the report](https://automattic.github.io/jetpack-e2e-reports/20273/report/) and you should find detailed test steps in for each test in the setup and test body sections.
